packages/xchain-thorchain/src/client.ts

```diff
@@ -389,9 +397,14 @@ class Client implements ThorchainClient, XChainClient {
         txMaxHeight,
       })
 
+      const txs: Txs = []
+      for (const tx of txHistory.txs) {
+        txs.push(await this.getTransactionData(tx.hash))
```

Comment:
To avoid many requests by default, set default values of [offset=0](https://github.com/xchainjs/xchainjs-lib/blob/86c13c20b4538ab14aadb09d1bf35d58154dd694/packages/xchain-thorchain/src/client.ts#L381) / [limit=10](https://github.com/xchainjs/xchainjs-lib/blob/86c13c20b4538ab14aadb09d1bf35d58154dd694/packages/xchain-thorchain/src/client.ts#L382) before.